test/guardian/plug/ensure_not_authenticated_test.exs

```diff
@@ -0,0 +1,95 @@
+defmodule Guardian.Plug.EnsureNotAuthenticatedTest do
+  use ExUnit.Case, async: true
+  use Plug.Test
+
+  alias Guardian.Plug.EnsureNotAuthenticated
+
+  defmodule TestHandler do
+    def authenticated(conn, _) do
+      conn
+      |> Plug.Conn.assign(:guardian_spec, :authenticated)
+      |> Plug.Conn.send_resp(401, "Authenticated")
+    end
+  end
+
+  test "init/1 sets the handler option to the module that's passed in" do
+    %{handler: handler_opts} = EnsureNotAuthenticated.init(handler: TestHandler)
+
+    assert handler_opts == {TestHandler, :authenticated}
+  end
+
+  test "init/1 defaults the handler option to Guardian.Plug.ErrorHandler" do
+    %{handler: handler_opts} = EnsureNotAuthenticated.init %{}
+
+    assert handler_opts == {Guardian.Plug.ErrorHandler, :authenticated}
+  end
+
+  test "init/1 with default options" do
+    options = EnsureNotAuthenticated.init %{}
+
+    assert options == %{
+      claims: %{},
+      handler: {Guardian.Plug.ErrorHandler, :authenticated},
+      key: :default
+    }
+  end
+
+  test "it validates claims and calls through if the claims are not ok" do
+    claims = %{ "aud" => "token", "sub" => "user1" }
```

Comment:
Hey @victorlcampos, for consistency sake could you remove the extra whitespace in your Maps?  It looks like it's this line and lines 46 and 54.

`%{ "aud" => "token", "sub" => "user1" }` -> `%{"aud" => "token", "sub" => "user1"}`

Thanks :+1: